examples/fitness/README.md

```diff
@@ -0,0 +1 @@
+# fitness
```

Comment:
Might as well right a realer README. :)  But OK.

Fitness is a demo which shows text input and saving/reading from a json file.  Honestly we could probably delete the fitness example at this point.  @Hixie